core/common/src/main/java/org/phenotips/remote/common/internal/api/DefaultPatientToJSONConverter.java

```diff
@@ -117,7 +117,7 @@ private static JSONObject contact(Patient patient)
             // TODO: replace this with a URL to a match/contact page
             List<String> email = contact.getEmails();
             if (!email.isEmpty() && !StringUtils.isBlank(email.get(0))) {
-                href += email.get(0) + ",matchmaker@phenomecentral.org";
+                href += email.get(0) + ", matchmaker@phenomecentral.org";
```

Comment:
we need to check if spaces are allowed by specification here, this change will not only affect our UI but MME output as well

Reply:
`href` : A public (no login required) URL for contacting the owner of the patient record to follow up with a match. This must be a valid URL (of the form `<scheme>:<address>`), and could take a number of forms: (Mandatory)
An `HTTP` URL: in this case, the URL could be a contact form which would allow the user to contact the owner of the matched patient.
A `mailto` URL: in this case, the URL could be a (potentially-anonymized) email address to contact regarding the patient match. It is preferred to use the email field for this purpose.
`email` : A (potentially-anonymized) email address for contacting the owner of the patient record to follow up with a match. (Optional) (since v1.1)